docs/workflows/index.rst

```diff
@@ -0,0 +1,39 @@
+Workflows
+=========
+
+If you have not yet installed the Python plugins on your Pulp installation, please follow our
+:doc:`../installation`. These documents will assume you have the environment installed and
+ready to go.
+
+The REST API examples here use `httpie <https://httpie.org/doc>`_ to perform the requests.
+The ``httpie`` commands below assume that the user executing the commands has a ``.netrc`` file
+in the home directory. The ``.netrc`` should have the following configuration:
+
+.. code-block:: bash
+
+    machine localhost
+    login admin
+    password admin
+
+If you configured the ``admin`` user with a different password, adjust the configuration
+accordingly. If you prefer to specify the username and password with each request, please see
+``httpie`` documentation on how to do that.
+
+To make these workflows copy/pastable, we make use of environment variables. The first variable to
+set is the hostname and port.::
+
+   $ export PULP_BASE=http://<hostname>:8000
```

Comment:
I suggest that this be named something different.  ```PULP_BASE``` sounds like like it would be the Pulp API root or something, i.e. ```http://<hostname>:8000/pulp/```

Maybe ```BASE_ADDR``` or ```ADDRESS```?  It's just a suggestion so if you disagree, I'm fine with not doing this.